services/api-server/src/simcore_service_api_server/models/schemas/profiles.py

```diff
@@ -17,10 +18,12 @@ class ProfileUpdate(ProfileCommon):
 
 
 class UserRoleEnum(StrAutoEnum):
+    # NOTE: this is in sync with simcore_postgres_database.models.users.UserRole via testing
     ANONYMOUS = auto()
     GUEST = auto()
     USER = auto()
     TESTER = auto()
+    PRODUCT_OWNER = auto()
```

Comment:
Q: what impact will this have on things like garbage collection and permissions. Have you thought about those?

Reply:
yes